pycoshark/mongomodels.py

```diff
@@ -539,7 +557,7 @@ class ActionRun(Document):
     status = StringField()
     conclusion = StringField()
     workflow_id = ObjectIdField(required=True)
-    pull_requests = ListField()
+    pull_requests = EmbeddedDocumentListField(RunPullRequest, default=[])
```

Comment:
Mutable default types like this are evaluated on definition time and may not behave as you would expect from other languages (see: https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).

Reply:
To be honest, I just revert the changes back to the old one. Because using PullRequest module instead of RunPullRequest did not workout. Anyway, I will remove this default because it's not making any difference.